pmix/borrow.py

```diff
@@ -31,12 +31,82 @@
 from pmix.xlsform import Xlsform
 
 
-def create_translation_dict(xlsxfile: List[str], correct: List[str]) \
+parser = argparse.ArgumentParser(
```

Comment:
Let's put this `ArgumentParser` in a function that returns the ArgumentParser. I prefer not to have the parser as a global variable in the module.